lighthouse/src/main.rs

```diff
@@ -165,6 +165,45 @@ fn main() {
                 )
                 .global(true),
         )
+        .arg(
+            Arg::with_name("terminal-total-difficulty-override")
+                .long("terminal-total-difficulty-override")
+                .value_name("INTEGER")
+                .help("Used to coordinate manual overrides to the TERMINAL_TOTAL_DIFFICULTY parameter. \
+                       This flag should only be used if the user has a clear understanding that \
+                       the broad Ethereum community has elected to override the terminal difficulty. \
+                       Incorrect use of this flag will cause your node to experience a consensus
+                       failure. Be extremely careful with this flag.")
+                .takes_value(true)
+                .global(true)
```

Comment:
This is going to make these flags available in `lighthouse account_manager` which is a little strange.  But I think this is fixable once we upgrade to `clap v3` (v3 is still in beta but I implemented the upgrade here: https://github.com/sigp/lighthouse/pull/2808)

Conversation about why it's not yet fixable:
https://github.com/clap-rs/clap/issues/1183

Reply:
Yeah, it's indeed a bit unfortunate. However, I think I'm OK with that. The ability to ensure a consistent spec across all components is worth the occasional weirdness. With this setup, at least the `account_manager` has access to a correct spec, if it turns out it needs it later.